packages/core-mobile/app/new/routes/(signedIn)/(tabs)/_layout.tsx

```diff
@@ -104,3 +110,98 @@ export default function TabLayout(): JSX.Element {
     </BottomTabs>
   )
 }
+
+const TabBar = ({
+  state,
+  descriptors,
+  navigation
+}: BottomTabBarProps): JSX.Element => {
+  const insets = useSafeAreaInsets()
+  const { theme } = useTheme()
+
+  const backgroundColor = useMemo(() => {
+    return theme.isDark
+      ? isIOS
+        ? alpha('#121213', 0.8)
```

Comment:
121213 is neutral950. looks like it is being used in a few places. we should create a const for it or import from k2.